backend/.env.example

```
@@ -9,6 +9,8 @@ DISCORD_BOT_TOKEN=
 # ENABLE_DISCORD_BOT=true
 
 GITHUB_TOKEN=
+# Add Org Name here
+GITHUB_DEFAULT_ORG=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The GITHUB_DEFAULT_ORG key should go before the GITHUB_TOKEN key

(UnorderedKey)